cob/cob.h

```diff
@@ -267,11 +269,11 @@ struct m0_cob_domain {
 	 * m0_be_btree has it's own volatile-only fields, so it can't be placed
 	 * before the m0_format_footer, where only persistent fields allowed
 	 */
-	struct m0_be_btree      cd_object_index;
-	struct m0_be_btree      cd_namespace;
-	struct m0_be_btree      cd_fileattr_basic;
+	struct m0_be_btree      cd_object_index[COB_HT_SIZE];
+	struct m0_be_btree      cd_namespace[COB_HT_SIZE];
+	struct m0_be_btree      cd_fileattr_basic[COB_HT_SIZE];
 	struct m0_be_btree      cd_fileattr_omg;
-	struct m0_be_btree      cd_fileattr_ea;
+	struct m0_be_btree      cd_fileattr_ea[COB_HT_SIZE];
```

Comment:
Please add comments to explain why the cd_fileattr_omg does not need to be an array.
* OMG will be relatively a small set of data. No need to be divided into multiple btree?